### PR TITLE
Fix numbered lists

### PR DESCRIPTION
**Pull request by cody-clark**
Preview: https://deploy-preview-10887--kubernetes-io-master-staging.netlify.com/docs/tutorials/hello-minikube/

### CODE REVIEW

File: content/en/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -79,51 +79,51 @@ recommended way to manage the creation and scaling of Pods.
...
 Pod runs a Container based on the provided Docker image. 
 
-  ```shell
-  kubectl run hello-node --image=gcr.io/hello-minikube-zero-install/hello-node --port=8080
-  ```
+    ```shell
+    kubectl run hello-node --image=gcr.io/hello-minikube-zero-install/hello-node --port=8080
+    ```
 
 2. View the Deployment:
```
Comment (tengqm) [inline content/en/docs/tutorials/hello-minikube.md]: Maybe we can change the above line to `1. View the Deployment` now?

File: content/en/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -134,124 +134,124 @@ Kubernetes [*Service*](/docs/concepts/services-networking/service/).
...
+    - kube-dns: enabled
+    - metrics-server: disabled
+    - nvidia-driver-installer: disabled
+    - nvidia-gpu-device-plugin: disabled
+    - registry: disabled
+    - registry-creds: disabled
+    - storage-provisioner: enabled
+    ```
+    {{< note >}}
+    **Note:**
```
Comment (stewart-yu) [inline content/en/docs/tutorials/hello-minikube.md]: Please fix here https://deploy-preview-10887--kubernetes-io-master-staging.netlify.com/docs/tutorials/hello-minikube/

File: content/en/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -79,51 +79,51 @@ recommended way to manage the creation and scaling of Pods.
...
+    ```
   
-  {{< note >}}
-  **Note:** For more information about `kubectl`commands, see the
-  [kubectl overview](/docs/user-guide/kubectl-overview/).
-  {{< /note >}}
+    {{< note >}}
+    **Note:** For more information about `kubectl`commands, see the
+    [kubectl overview](/docs/user-guide/kubectl-overview/).
+    {{< /note >}}
```
Comment (stewart-yu) [inline content/en/docs/tutorials/hello-minikube.md]: and here

File: content/en/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -48,20 +48,20 @@ For more information, read the Docker documentation for [docker build](https://d
...
-   ```shell
-   minikube start
-   ``` 
-   {{< /note >}}
+    {{< note >}}
+    **Note:** If you installed Minikube locally, run
+    ```shell
+    minikube start
+    ``` 
+    {{< /note >}}
```
Comment (stewart-yu) [inline content/en/docs/tutorials/hello-minikube.md]: The same

**Pull request merged**